introduction.md

```diff
@@ -1,2 +1,4 @@
 Hi this is srikanth katragadda. I am excited to learn GitHub
 My favourite codeschool path is: https://www.codeschool.com/paths/javascript
+a* JAVA
+a* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?